### DISCUSSION TITLE
Agent clean-up!

### DISCUSSION THREAD

**Discussion by mkoo** [ArctosDB/arctos#6136] [Agent Committee]
Just a quick note that I have a student Jihyun Kim helping clean-up [the low-data agents from this list](https://docs.google.com/spreadsheets/d/11Hy5kG7kcrSIBB-VLY9jedRp5jDWGCZlJKnY-jKv0X0/edit#gid=769806014). She's focused on the MVZ folks and adding in info from remarks (like degree dates and major advisor), birth/death dates if she can find. She's marking the Agentified column with yes or no.

I also added a new tab for Resources with a list of websites that may be useful to confirm people and find biographical details to help figure our list of names. 

Exciting to see progress!

**Comment by lin-fred** [comment]
Awesome thanks!
